Drop-In/src/main/res/layout/bt_fragment_select_payment_method_parent.xml

```diff
@@ -16,6 +16,17 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <Button
```

Comment:
I think for accessibility and to align with iOS instead of clicking outside to cancel we should add a cancel button. Here's an [iOS PR](https://github.com/braintree/braintree-ios-drop-in/pull/279) that shows the refactoring for this.

Reply:
I think this may be Android UX/visual territory, maybe we can table for now? The screen reader will announce this as a button for now. From an accessibility standpoint we should be good.

Reply:
Yeah we can revisit that with additional UI changes later 👍